### ISSUE TITLE
Choosing a good identity_loss_weight

### ISSUE THREAD

**Issue by JustABunchOfCode** [LynnHo/CycleGAN-Tensorflow-Simple#25]
Is 0.0 a good choice for a default value? 

Regarding to the paper, the default should be 0.5, shouldn't it?

Code: py.arg('--identity_loss_weight', type=float, default=0.0)

**Comment by LynnHo** [comment]
@JustABunchOfCode For different datasets, it seems to be different default values.

**Comment by JustABunchOfCode** [comment]
@LynnHo 
Can you tell me a useful input range of the "identity_loss_weight"? 
Is its meaning equal to the original paper / reference implementation as a scaling pre-factor of lambda in [0,1] or is it an independent absolute weight?

**Comment by LynnHo** [comment]
@JustABunchOfCode According to the Cyclegan paper, the weight for the identity mapping loss was 0.5λ where λ was the weight for cycle consistency loss (for Monet’s paintings→photos and for flower photo enhancement). But I am not sure which value is more generalizable for different datasets.

**Comment by JustABunchOfCode** [comment]
@LynnHo 
regarding to the Monet’s paintings→photos example: Should the identity loss weight parameter 0.5 or 5? Because the reference implementation in PyTorch expects the factor 0.5 and I think your implementation expects the result 5, doesn't it?

**Comment by LynnHo** [comment]
@JustABunchOfCode It should be set as 5 in this code.

**Comment by JustABunchOfCode** [comment]
@LynnHo okay that confused me a bit. Thank you very much. Maybe you could add a training example with weights to the ReadMe?
